packages/components/src/components/Dropdown/Dropdown.tsx

```diff
@@ -94,6 +94,11 @@ export type DropdownContentProps = {
      * Prevents the focus from being set on the trigger when the dropdown is closed.
      */
     preventTriggerFocusOnClose?: boolean;
+    /**
+     * Define the minimum distance between the dropdown and the viewport edge
+     * @default 8
```

Comment:
Default 8 from 'default' | 'large'?

```suggestion
     * @default 'default'
```